### DISCUSSION TITLE
2019/02/08/how-to-be-a-qualified-developer

### DISCUSSION THREAD

**Discussion by giscus** [kaochenlong/kaochenlong-discuss#14] [General]
# 2019/02/08/how-to-be-a-qualified-developer

「我不是本科生，適合寫程式嗎？」這樣的討論，是每個想轉職軟體工程師的人心中，一定曾冒出來的問題。這篇文章就帶你了解，成為軟體工程師你必須經歷的三個階段。

https://kaochenlong.com/2019/02/08/how-to-be-a-qualified-developer.html

**Comment by LIUWENZHENG12** [comment]
謝謝您的分享，願意把軟體工程師的3個階段說的深入淺出，我放棄當軟體工程師了
